packages/um7/package.py

```diff
@@ -28,38 +25,34 @@ class Um7(Package):
 
     maintainers("penguian")
 
-    depends_on("dummygrib", type=("build", "link"))
     depends_on("fcm", type="build")
+    depends_on("dummygrib", type=("build", "link"))
     depends_on("gcom4@access-esm1.5+mpi", type=("build", "link"))
     depends_on("openmpi@4.0.2:4.1.0", type=("build", "run"))
-    depends_on("netcdf-fortran@4.5.1:4.5.2", type=("build", "link"))
+    depends_on("netcdf-fortran@4.5.2", type=("build", "link"))
     depends_on("oasis3-mct@access-esm1.5", type=("build", "link"))
 
-    variant("omp", default=True, description="Use OpenMP")
-    variant("netcdf", default=True, description="NetCDF")
     variant("opt", default="high", description="Optimization level",
```

Comment:
Can variant "opt" be renamed to something that indicates 'optimisation'?

Reply:
Changed to `"optim"`.